hopp/sites/site_info.py

```diff
@@ -127,6 +131,13 @@ def __init__(self, data,
                                             filepath=wind_resource_file)
             self.n_timesteps = 8760
 
+        if 'no_wave' not in data:
```

Comment:
How about this?
```
if 'no_wave' not in data and not data['no_wave']:
   data['no_wave'] = False
else:
     self.wave_resource = WaveResource(data['lat'], data['lon'], data['year'], filepath = wave_resource_file)
     self.n_timesteps = 8760
```

Reply:
What about changing it to 
`''solar'': bool (optional) if ''False'' solar data download for site is skipped, otherwise solar resource is downloaded from NSRDB
``no_wind``: bool (optional), if ``True`` wind data download for site is skipped, otherwise wind resource is downloaded from wind-toolkit
``no_wave``: bool (optional), if ``True`` wave data download for site is skipped, otherwise wave resource is downloaded from wind-toolkit


if data['wave'] == False:
        pass
else:
        self.wave_resource = WaveResource(data['lat'], data['lon'], data['year'], filepath = wave_resource_file)
        self.n_timesteps = 8760
`

Reply:
Yeah I think it's a good idea to make the these solar, wind and wave optional inputs be about downloading files, and leaving the "x_resource_file" as is.

So if a solar_resource_file doesn't exist and the solar download input is False, then don't download and don't make a solar_resource.

I think in that case your documentation makes sense but the names should probably be `download_solar` or something so that it's clear what it does.

and the logic would be:

```
if data['wave_download'] and len(wave_resource_file):
        raise ValueError("Wave resource file was provided but option to download wave resource was selected. To download, do not provide wave resource file. To use a wave resource file, set `wave_download` to False")
elif data['wave_download'] or len(wave_resource_file):
    self.wave_resource = WaveResource(data['lat'], data['lon'], data['year'], filepath = wave_resource_file)
    self.n_timesteps = 8760

```

Reply:
Changes in https://github.com/NREL/HOPP/pull/133/commits/1875752b53330ee0e209ca29478992aa587d7727 ended up being a little different. wave_resource_file is a path so it doesn't have length. Open to other ideas too